suite/tests/api/ir_x86.c

```diff
@@ -111,10 +111,9 @@ static byte buf[8192];
 #define X86_ONLY 1
 #define X64_ONLY 2
 
-static void
-test_all_opcodes_0(void *dc)
-{
-#define INCLUDE_NAME "ir_x86_0args.h"
+/***************************************************************************/
+/*********************** OPCODE_FOR_CREATE 0 args **************************/
```

Comment:
Use the banner style in the style guide, here and below.

Reply:
Done.